### PR TITLE
Update README.md for clarity and concision

### PR DESCRIPTION
**Pull request by b1skit**
Reworded several sentences and paragraphs for clarity and concision. No key
information or links were altered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,20 @@ Homepage: https://godotengine.org
...
 #### Free, open source and community-driven
 
 Godot is completely free and open source under the very permissive MIT license.
-No strings attached, no royalties, nothing. The users' games are theirs, down
+No strings attached, no royalties, nothing. Godot users' games are theirs, down
 to the last line of engine code. Godot's development is fully independent and
-community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the Software Freedom Conservancy
-not-for-profit.
+community-driven, and supported by the Software Freedom Conservancy. Users are
```
Comment (akien-mga) [inline README.md]: Not sure the rephrasing adds clarity here, the "empowerment" derives from the fact that Godot's development is independent and community-driven (i.e. users can make changes). Splitting it in its own phrase after mentioning Conservancy breaks this link. I'd keep the original phrasing and just remove the "help" word:

> Godot's development is fully independent and community-driven, empowering users to shape the engine to match their expectations. Godot is supported by the Software Freedom Conservancy not-for-profit charity.

File: README.md [hunk 2]
```diff
@@ -32,37 +31,36 @@ years as an in-house engine, used to publish several work-for-hire titles.
...
-To get in touch with the developers, the best way is to join the
+The best way to get in touch with the Godot developers is to join the
 [#godotengine IRC channel](https://webchat.freenode.net/?channels=godotengine)
 on Freenode.
 
 ### Documentation and demos
 
 The official documentation is hosted on [ReadTheDocs](http://docs.godotengine.org).
-It is maintained by the Godot community in its own [GitHub repository](https://github.com/godotengine/godot-docs).
+It is maintained by the Godot community on its own [GitHub repository](https://github.com/godotengine/godot-docs).
```
Comment (akien-mga) [inline README.md]: Here you change "in" for "on", but not in the next sentence about the demos repo.

**Comment by groud** [comment]
I'm not really convinced that adding "Godot" in all those sentences makes things clearer. We know we are a talking about Godot, it's its Readme.md file.

File: README.md (hunk 2 above)
Comment (b1skit) [reply README.md]: Great catch, thank you!

File: README.md (hunk 1 above)
Comment (b1skit) [reply README.md]: That's a good point, thank you.

**Comment by b1skit** [comment]
Hi all, thank you for your feedback. I've made more updates based on your comments.

**Comment by b1skit** [comment]
@akien-mga @groud Were there any other changes you guys would like to see before approving the pull request?

File: README.md [hunk 3]
```diff
@@ -6,21 +6,20 @@ Homepage: https://godotengine.org
...
 #### 2D and 3D cross-platform game engine
 
-Godot Engine is a feature-packed, cross-platform game engine to create 2D and
-3D games from a unified interface. It provides a comprehensive set of common
-tools, so that users can focus on making games without having to reinvent the
-wheel. Games can be exported in one click to a number of platforms, including
-the major desktop platforms (Linux, Mac OSX, Windows) as well as mobile
-(Android, iOS) and web-based (HTML5) platforms.
+Godot Engine is a feature-packed, cross-platform game engine used to create 2D
+and 3D games from a unified interface. Godot provides a comprehensive set of
```
Comment (groud) [inline README.md]: I would keep "It" here

File: README.md [hunk 4]
```diff
@@ -6,21 +6,20 @@ Homepage: https://godotengine.org
...
 
 Godot is completely free and open source under the very permissive MIT license.
-No strings attached, no royalties, nothing. The users' games are theirs, down
+No strings attached, no royalties, nothing. Godot users' games are theirs, down
 to the last line of engine code. Godot's development is fully independent and
-community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the Software Freedom Conservancy
-not-for-profit.
+community-driven, empowering users to shape the engine to their expectations.
+Godot is supported by the Software Freedom Conservancy not-for-profit charity.
```
Comment (groud) [inline README.md]: I would keep "It" here too. Else you have 4 times Godot at the beginning of a sentence in one paragraph, which is quite a heavy style

File: README.md [hunk 5]
```diff
@@ -32,37 +31,35 @@ years as an in-house engine, used to publish several work-for-hire titles.
 
 #### Binary downloads
 
-Official binaries for the Godot editor and the export templates can be found
+Official binaries for the Godot editor and export templates can be found
 [on the homepage](https://godotengine.org/download).
 
-#### Compiling from source
+#### Compiling Godot from source
```
Comment (groud) [inline README.md]: Not sure it's needed here too

**Comment by groud** [comment]
I think almost all the "Godot" you added should actually be removed. It make the writting style heavy.
Replacing "It" by "Godot" does not really improve the consicion.

But to be honest, I'm not even conviced this PR improves the text at all. I think it was already clear before, the new text is still clear but just written in another way.

File: README.md (hunk 4 above)
Comment (b1skit) [reply README.md]: That's a good point, thanks.

File: README.md (hunk 5 above)
Comment (b1skit) [reply README.md]: Yeah - I agree it's inconsistent with the other section headers. I'll remove it.

**Comment by b1skit** [comment]
Thanks for the new feedback @groud. I've made the changes you mentioned, I appreciate the input.

I do believe the PR improves the text in multiple ways. My goal was to maintain all of the original information, and just polish the writing a bit. For example, some sentences were fragmented by missing words, and others contained incorrect use of definite articles, there was some missing punctuation etc. All of this stuff is relatively minor, but summed together I thought it was worth the effort.

**Comment by groud** [comment]
It looks ok to me now. If other people are ok with the changes we can merge.

**Comment by groud** [comment]
Oh sorry, you will have to [squash the commit together](http://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html?highlight=squash#mastering-the-pr-workflow-the-rebase) though.

**Comment by b1skit** [comment]
Hey @groud , I've squashed the commit together into a single commit as per your request. Let me know if there's anything else you need.

Thanks :)

**Comment by akien-mga** [comment]
To be honest I still agree with @groud, I don't see why repeating "Godot" in every single sentence makes the readme clearer. It's the readme file of the `godotengine/godot` repository, which is titled `## Godot Engine` at the top. I don't see why we'd need to be more explicit than that.

The section `### Documentation and demos` now mentions Godot 5 times (once per sentence), against only once previously.

In `#### Free, open source and community-driven` we already mentioned Godot twice, no need to repeat it a third time.

**Comment by b1skit** [comment]
Hi @akien-mga , thanks for replying back. 

The `### Documentation and demos` section originally started by using definite articles as referents to Godot, without ever specifically mentioning Godot by name. If we're being nit-picky, the sentences '**The** official documentation...' and '**It** is maintained...'  suggest that there is only one single documentation object in existence, which is incorrect. This section contains a list of single sentences that serve as (very short) paragraphs, and each paragraph requires a clearly defined topic.

In the `#### Free, open source and community-driven` section, I believe that each of the 3 mentions of Godot is appropriate. The first usage defines the subject of the opening sentence. The second mention follows a sentence discussing "users' games", so it is necessary to clarify the subject. The third usage follows a discussion of "empowering users" so it is again necessary to clarify the subject. If you read this section aloud, it flows well and makes sense.

That being said, I totally agree that in the context of the README.md that we all know we're talking about Godot. I think we could go either way on these points and the document wouldn't be any more or less readable. I do think that overall the proposed changes are positive. Now that 3 revisions have been made and the commits have been squashed together, I was hoping the changes could be accepted?

**Comment by groud** [comment]
> Now that 3 revisions have been made and the commits have been squashed together, I was hoping the changes could be accepted?

Well, since we are the two of us not really convinced by the changes, it might need a proper PR review. I guess we can decide in a PR meeting what to do with it. Please don't be offended if this PR get refused, the README.md file is a little bit our showcase on Github's, I believe it is understandable that we do not accept changes easily. :)

**Comment by mhilbrunner** [comment]
Closing this as there is no support for merging the PR.

Sorry, we should have caught this sooner and not asked you to spend further time on edits.
We still thank you for spending the time helping to improve Godot!

**Pull request closed without merging**